File: Nicollas.Dto/EvaluationDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Nicollas.Dto
{
    public class EvaluationDto
    {
        public int SubmittedAnswerId { get; set; }
        public DateTime SubmitDateTime { get; set; }
        public bool Correct { get; set; }
        public int Progress { get; set; }
        public int UserId { get; set; }
        public int ExerciseId { get; set; }
        public string Difficulty { get; set; }
        public string Subject { get; set; }
        public string Domain { get; set; }
        public string LearningObjective { get; set; }
    }
}

File: Nicollas.Web/Angular/src/app/Dto/EvaluationDto.ts
/*
* This template uses the TsT - Typewriter.
* Read more at :
* -- https://github.com/frhagn/Typewriter (Github)
* -- https://marketplace.visualstudio.com/items?itemName=frhagn.Typewriter  (Extension of Visual Studio 17)
* -- http://frhagn.github.io/Typewriter/ (Documentation)
*/

/* tslint:disable*/

declare namespace Nicollas.Dto {

	interface evaluationDto  { 
        submittedAnswerId: number;
        submitDateTime: Date;
        correct: boolean;
        progress: number;
        userId: number;
        exerciseId: number;
        difficulty: string;
        subject: string;
        domain: string;
        learningObjective: string;
    }
}